src/test/java/com/tradeshfit/dao/JdbcMessagesDAOTest.java

```diff
@@ -0,0 +1,99 @@
+package com.tradeshfit.dao;
+
+import com.tradeshfit.model.dao.JdbcMessagesDAO;
+import org.junit.Assert;
+import org.junit.Test;
+
+import javax.sql.DataSource;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.util.Date;
+
+import static org.mockito.Mockito.*;
+
+public class JdbcMessagesDAOTest {
+
+    @Test
+    public void TestInsertUseDataSource(){
+        try {
+            DataSource dataSource = mock(DataSource.class);
+            Connection connection = mock(Connection.class);
+            PreparedStatement preparedStatement = mock(PreparedStatement.class);
+
+            when(dataSource.getConnection()).thenReturn(connection);
+            when(connection.prepareStatement(any(String.class))).thenReturn(preparedStatement);
+
+            JdbcMessagesDAO dao = new JdbcMessagesDAO(dataSource);
+            Date date = new Date();
+            dao.insert("Kasper", date);
+            verify(dataSource, times(1)).getConnection();
+            verify(connection, times(1)).prepareStatement(any(String.class));
+            verify(preparedStatement, times(1)).setString(1, "Kasper");
+            verify(preparedStatement, times(1)).executeUpdate();
+            verify(preparedStatement, times(1)).close();
+            verify(connection, times(1)).close();
+        } catch (Exception exp){
+            Assert.assertTrue(false);
```

Comment:
`Assert.assertTrue(false)` -> `Assert.fail()`
However, any thrown exception will make the test method fail, so this try/catch is actually redundant.